grails-app/i18n/messages_de.properties

```diff
@@ -246,6 +246,7 @@ upload.lists.reload.header=Eine existierende Liste neu laden
 upload.lists.checkdata.errormessage=Ungültiger Dateityp: nur TAB- bzw. Komma-getrennte Textdateien sind erlaubt!
 
 permissions.lists.text=In diesem Formular kann der/die AutorIn der Liste (oder AdministratorIn) weitere BenutzerInnen als <strong>EditorInnen</strong> dieser Liste hinzufügen bzw. löschen. Diese <strong>EditorInnen</strong> dürfen danach ebenfalls die Einträge in der Liste ändern bzw. löschen.
+permissions.lists.warning=Bitte beachten Sie, dass die eingegebene E-Mail-Adresse, wenn sie nicht mit einem vorhandenen ALA-Benutzer �bereinstimmt, beim Speichern nicht zur Berechtigungsliste hinzugef�gt wird.
```

Comment:
You don't need to do non-EN translations - the LA team does this using external i18n files and Vicente monitors commits and adds them to a separate repo.